packages/auth0-ai/auth0_ai/authorizers/ciba/ciba_authorizer_params.py

```diff
@@ -3,13 +3,18 @@
 from auth0_ai.authorizers.types import ToolInput
 from auth0_ai.stores import Store
 
+
 class CIBAAuthorizerParams(TypedDict, Generic[ToolInput]):
     """
     Authorize Options to start CIBA flow.
 
     Attributes:
         scope (list[str]): The scopes to request authorization for.
         binding_message (Union[str, Callable[..., Awaitable[str]]]): A human-readable string to display to the user, or a function that resolves it.
+        authorization_details (Optional[Union[list[dict], Callable[..., Awaitable[list[dict]]]]]):
+            Authorization details that specify what the user is authorizing. Can be:
```

Comment:
link to a doc showing the expected structure (or sample) of the authorization_details object